### ISSUE TITLE
`[` should trigger indention

### ISSUE THREAD

**Issue by lorenzwalthert** [krlmlr/styler#206]
Please indent the second line:
``` r
styler::style_text(                                  
'ranges[tag == "non_literal" & str_detect(text, ";"),
text := str_replace_all(text, ";", "\n")]'          
)                                                    
#> ranges[tag == "non_literal" & str_detect(text, ";"),
#> text := str_replace_all(text, ";", "\n")]
```

Looks like `[` could simply be added to the list of `indent_op()`. RStudio does not indent token dependent, so I guess a simple solution is fine.
